services/test-results.ts

```diff
@@ -36,16 +36,30 @@ function getTestResultFromDay(enrichedDay: ReturnType<typeof enrichDayWithData>,
   const templateResult = enrichedDay.templates.find((it) => it.template === template);
   if (templateResult == null || ciLink == null) return { status: 'no-data', date: enrichedDay.date, ciLink };
 
+  const metadata = getMetadata(enrichedDay.pipeline, template);
+
   const features = templateResult.features;
   return {
     date: enrichedDay.date,
     ciLink,
-    storybookVersion: undefined,
+    storybookVersion: metadata?.version,
     features: features,
     status: getStatusOfTestResult(features),
   };
 }
 
+function getMetadata(pipeline: EnrichedPipeline, template: string): { version: string } | undefined {
+  const buildJob = pipeline.jobs.find((it) => it.name === 'build-sandboxes');
+  if (buildJob == null) return;
+  const metaDataTest = buildJob.tests.find((it) => it.name.includes(`${template} - metadata`));
+  if (metaDataTest == null) return;
+  try {
+    return JSON.parse(metaDataTest.message);
+  } catch (e) {
+    return;
```

Comment:
I assume we want to throw the error? but just in case it's actually an invalid JSON

Reply:
I actually just gonna bubble up this error, as we really should not get it.